godtools/Views/Cells/HomeToolTableViewCell.swift

```diff
@@ -77,12 +80,13 @@ class HomeToolTableViewCell: UITableViewCell {
         
         configureParallelLanguageLabel(parallelLanguage: parallelLanguage)
 
-        numberOfViewsLabel.text = String.localizedStringWithFormat("total_views".localized, resource.totalViews)
+        tractDescriptionLabel.text = loadDescription(resource: resource)
     }
     
     private func configureParallelLanguageLabel(parallelLanguage: Language?) {
         if resource != nil && resource!.isAvailableInLanguage(parallelLanguage)  {
-            languageLabel.text = parallelLanguage!.localizedName()
+            let check: String = "✓ "
+            languageLabel.text = check + parallelLanguage!.localizedName()
```

Comment:
I know this was in the code before you touched it, but could you refactor this line and line 87 to use `guard let` statements to unwrap those optionals?